euctr/crawl/tests/test_scraper.py

```diff
@@ -0,0 +1,94 @@
+import tempfile
+import psycopg2
+
+from django.test import SimpleTestCase
+import datetime
+import shutil
+import testing.postgresql
+from pathlib import Path
+import subprocess
+import psycopg2.extras
+
+
+def crawl_report(db, registry_id):
+    # We run the crawler in a subprocess, as this is the neatest way
+    # to deal with restarting a Twisted reactor..
+    with tempfile.TemporaryFile('r+') as f:
+        subprocess.check_call(
+            ['./manage.py',
+             'run_crawler',
+             '--query=' + registry_id,
+             '--config=crawl.base.test_config',
+             '--db=' + db],
+            stdout=f,
+            stderr=f
+        )
+        f.seek(0)
+        output = f.read()
+        assert "['cached']" in output, output + "\n(should be a fixture)"
```

Comment:
This line doesn't require `f` to be open, so undent it.  There's only one line here, but elsewhere in the code we have lots of excess indentation in situations like this.